i18n/locales/ar-SA/dashboard/courses.yml

```diff
@@ -1,5 +1,4 @@
 #Autogenerated scripts locale file.
-#---
```

Comment:
I'm not entirely sure why this line got removed (and it seems to have happened in all languages), but given that this is YAML and this line is irrelevant, I'm inclined to ignore it